src/screens/select.tsx

```diff
@@ -1,51 +1,12 @@
-/* eslint-disable @typescript-eslint/no-misused-promises */
```

Comment:
This is pretty huge, I think you should split it up into some subcomponents. For example you could have a component for increasing/decreasing a number with a label, and use it twice here, one for adults and one for children. You could also wrap your datepicker into a component and move that logic there.

Reply:
Will do this in the next refactor